README.md

```diff
@@ -2,9 +2,9 @@
-|**Release**|[![Build and Innerloop test status](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/windows_release.svg?label=build+%26+innerloop+test)](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/windows_release/) [![Outerloop test status](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/outerloop_windows_release.svg?label=outerloop+tests)](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/outerloop_windows_release/)|[![Build and Innerloop test status](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/linux_release.svg?label=build+%26+innerloop+test)](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/linux_release/)|
-|**Coverage Report**|[![Coverage Status](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/code_coverage_windows/lastStableBuild/Code_Coverage_Report/badge_combined.svg)](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/code_coverage_windows/)||
+|**Debug**|[![Build and Innerloop test status](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/master/windows_debug.svg?label=build+%26+innerloop+test)](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/master/job/windows_debug/) [![Outerloop test status](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/master/outerloop_windows_debug.svg?label=outerloop+tests)](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/master/job/outerloop_windows_debug/)|[![Build and Innerloop test status](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/master/linux_debug.svg?label=build+%26+innerloop+test)](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/master/job/linux_debug/)|
+|**Release**|[![Build and Innerloop test status](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/master/windows_release.svg?label=build+%26+innerloop+test)](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/windows_release/) [![Outerloop test status](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/master/outerloop_windows_release.svg?label=outerloop+tests)](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/master/job/outerloop_windows_release/)|[![Build and Innerloop test status](https://img.shields.io/jenkins/s/http/dotnet-ci.cloudapp.net/job/dotnet_wcf/master/linux_release.svg?label=build+%26+innerloop+test)](http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/master/job/linux_release/)|
```

Comment:
Link for Release/Windows/build+innerloop incorrect (should be /job/master/job/windows_release)